python/fbprophet/forecaster.py

```diff
@@ -198,21 +198,30 @@ def validate_column_name(self, name, check_holidays=True,
         reserved_names.extend([
             'ds', 'y', 'cap', 'floor', 'y_scaled', 'cap_scaled'])
         if name in reserved_names:
-            raise ValueError('Name "{}" is reserved.'.format(name))
+            raise ValueError(
+                'Name "{name}" is reserved.'.format(name=name)
```

Comment:
Perhaps rewrite as:
```suggestion
                'Name {name!r} is reserved.'.format(name=name)
```

Reply:
I did not know about this option! Changing the formats now....

Reply:
We learn new things each day. Im glad you learned that today.